From f0d7dab3aca40a3f861ac182f8f3eb2bbbf89631 Mon Sep 17 00:00:00 2001
From: Martin Hauke <mardnh@gmx.de>
Date: Thu, 3 Sep 2015 20:52:49 +0200
Subject: [PATCH] GCC 5.x inline function semantics fix

The compiler will warn and the linker will fail if an inline
function is declared but not defined in the same translation
unit. This fixes the problem by prepending 'extern' to the
declaration in the header file.

For a proper explanation see:
https://gcc.gnu.org/gcc-5/porting_to.html
---
 bitstream.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thirdparty/toolame-02l/bitstream.h b/thirdparty/toolame-02l/bitstream.h
index 66f8ae4..db67ac5 100644
--- a/thirdparty/toolame-02l/bitstream.h
+++ b/thirdparty/toolame-02l/bitstream.h
@@ -9,7 +9,7 @@ unsigned int get1bit (Bit_stream_struc *);
 void put1bit (Bit_stream_struc *, int);
 unsigned long look_ahead (Bit_stream_struc *, int);
 unsigned long getbits (Bit_stream_struc *, int);
-INLINE void putbits (Bit_stream_struc *, unsigned int, int);
+extern INLINE void putbits (Bit_stream_struc *, unsigned int, int);
 void byte_ali_putbits (Bit_stream_struc *, unsigned int, int);
 unsigned long byte_ali_getbits (Bit_stream_struc *, int);
 unsigned long sstell (Bit_stream_struc *);